packages/web-components/src/components/ic-radio-group/ic-radio-group.stories.mdx

```diff
@@ -140,7 +156,11 @@ import readme from "./readme.md";
             .querySelector("ic-radio-group")
             .addEventListener("icChange", handleIcChange);
         </script>
-        <ic-radio-group label="Conditional dynamic" name="1">
+        <ic-radio-group
+          label="Conditional dynamic"
+          orientation="horizontal"
```

Comment:
Should the horizontal orientation be removed from this story? My thinking is that the horizontal overide isn't being tested in this one, it's all about the Conditional rendering